modules/File/app/Services/StorageManagerService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Modules\File\App\Services;
+
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use Modules\File\App\Contracts\Services\StorageManagerInterface;
+
+class StorageManagerService implements StorageManagerInterface
+{
+    public function putFile($fileName, UploadedFile $file, string $path, $isPublic = false)
+    {
+        $savedPath = $this->getDisk($isPublic)->putFileAs($path, $file, $fileName);
```

Comment:
setDisk method
$disk property => set in setDistk()
when call getDisk , return disk property .